Common/src/CConfig.cpp

```diff
@@ -7604,6 +8557,8 @@ CConfig::~CConfig(void) {
     delete[] Marker_Supersonic_Outlet;
               delete[] Marker_Outlet;
           delete[] Marker_Isothermal;
+  delete[] Marker_IsothermalCatalytic;
+  delete[] Marker_IsothermalNonCatalytic;
```

Comment:
I'm not seeing to *** type containers being deleted here, are they deleted somewhere else?